frontend/src/app/okrview/okr-child-unit/department/department-tab-taskboard/department-tab-taskboard.component.html

```diff
@@ -16,11 +16,8 @@
             </mat-button-toggle>
         </mat-button-toggle-group>
     </div>
-    <div class="text-center add-button" *ngIf="isInteractive">
-        <button mat-mini-fab color="accent" (click)="onTaskAddButtonClick()">
-            <mat-icon matTooltip="{{ 'department-tab-taskboard.tooltip.create-new-task' | translate }}">add_circle_outline
-            </mat-icon>
-        </button>
+    <div class="text-center" *ngIf="isInteractive">
```

Comment:
Könnte hier die div-Box mit in den app-add-button tag integriert werden

Reply:
Frage bleibt bestehen